### PR TITLE
[FIX]Hide the Project Module

### PR DESCRIPTION
**Pull request by jananjoy**

### CODE REVIEW

File: project_event/views/project_project_view.xml [hunk 1]
```diff
@@ -23,6 +23,15 @@
               name="Configuration"
               parent="project_event_menu_root"
               groups="group_project_event_manager"/>
+
```
Comment (rimbendhaou) [inline project_event/views/project_project_view.xml:26]: Make this menu invisible in the project_specific

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/views/project_project_view.xml (hunk 1 above)
Comment (eilst) [reply project_event/views/project_project_view.xml:26]: Yes, this should go on gitlab

Comment (jananjoy) [reply project_event/views/project_project_view.xml:26]: resolved all the comments

Comment (jananjoy) [reply project_event/views/project_project_view.xml]: This functionality is implemented and already merged in another branch.

**Comment by jananjoy** [comment]
This functionality is implemented and already merged in another branch.

**Pull request closed without merging**